### ISSUE TITLE
BUG: A blank `~/.jobflow.yaml` file causes a crash

### ISSUE THREAD

**Issue by arosen93** [materialsproject/jobflow#411]
If the user has a `~/.jobflow.yaml` file and it's blank, the following error is raised:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/rosen/software/miniconda/envs/quacc/lib/python3.10/site-packages/jobflow/__init__.py", line 14, in <module>
    SETTINGS = JobflowSettings()
  File "pydantic/env_settings.py", line 39, in pydantic.env_settings.BaseSettings.__init__
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for JobflowSettings
__root__
  'NoneType' object is not iterable (type=type_error)
```

**Comment by mcgalcode** [comment]
What do you think of leaving a warning for the user? https://github.com/materialsproject/jobflow/pull/415

**Comment by arosen93** [comment]
@mcgalcode: I think it would be cleaner if the presence of a blank YAML simply was interpreted as `{}` and didn't update any of the default parameters.

**Comment by mcgalcode** [comment]
I could raise a RuntimeError or SyntaxError for a malformed file and do nothing or leave a warning for empty an empty file.

**Comment by arosen93** [comment]
@mcgalcode: I think something like that might be a good solution! I don't personally know the best error to raise (that always confuses me...).

**Comment by mcgalcode** [comment]
Alright, updated the PR to warn if it's empty and raise a `ValueError` with a more descriptive message if it's badly formatted.

**Comment by arosen93** [comment]
Cool! Thanks! Once merged, we'll probably need to make the same change to Atomate2 😄

**Comment by utf** [comment]
Fixed in #415
